airflow/config_templates/default_airflow.cfg

```diff
@@ -208,6 +208,11 @@ max_num_rendered_ti_fields_per_task = 30
 # On each dagrun check against defined SLAs
 check_slas = True
 
+# Custom user function that returns a context manager. Syntax is "package:method".
+# Context is entered when operator starts executing task. __enter__() will be called
+# before the operator's execute method, and __exit__() shortly after.
+user_defined_execute_context =
```

Comment:
similar to what I mentioned in xcom_backend. Can we set the default value here as well for visibility?

Reply:
Also, I would simplify it to `execute_context`

Reply:
If we define a default - what would it be?
Just adding a syntax example like `module.function` would result in errors if the user doesn't set his own user context, or just blanks out the default.
What do you think?

Reply:
It should default to a method within Airfow. We should have a defult context provider.

Reply:
Added :)